app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,5 @@
-$govuk-compatibility-govukfrontendtoolkit: true;
 $govuk-compatibility-govuktemplate: true;
+$govuk-use-legacy-palette: false;
```

Comment:
Can we just delete this line rather than setting to false?

Reply:
nein. `$govuk-compatibility-govuktemplate: true;` in addtiion to other things turn on old colour, so we need the `$govuk-use-legacy-palette: false;` to use the new colours, but keep the rest of the compatibility settings

Reply:
Ah, I wondered if it was something like that. Makes sense, thanks for explaining!

Reply:
update facet tag button focus outline